src/main/java/seedu/address/logic/commands/note/NoteAddCommand.java

```diff
@@ -48,6 +49,10 @@ public CommandResult execute(Model model) throws CommandException {
     public CommandResult execute(SeplendidModel seplendidModel) throws CommandException {
         requireNonNull(seplendidModel);
 
+        if (seplendidModel.hasNote(noteToAdd)) {
```

Comment:
good catch on duplicate content checking. perhaps could add some tests in future